src/runtime/tests/refiner-test.ts

```diff
@@ -427,7 +461,7 @@ describe('SQLExtracter', () => {
       `);
       const schema = manifest.particles[0].handleConnectionMap.get('input').type.getEntitySchema();
       const query: string = SQLExtracter.fromSchema(schema, 'table');
-      assert.strictEqual(query, 'SELECT * FROM table WHERE ((a + (b / 3)) > 100) AND ((a > 3) AND (NOT (a = 100))) AND ((b > 20) AND (b < 100));');
+      assert.strictEqual(query, `SELECT * FROM table WHERE ((a + (b * ${1/3})) > 100) AND ((a > 3) AND (NOT (a = 100))) AND ((b > 20) AND (b < 100));`);
```

Comment:
It might be nice to support division still (if there's time later)

Reply:
The reason for that normalization is because of the following:
```
static divide(a: Fraction, b: Fraction): Fraction {
    **const invB = new Fraction(b.den, b.num);**
    return Fraction.multiply(a, invB);
  }
```
i.e. 3 is essentially getting flipped and converted to a new fraction (1/3). However the issue is the reduce function that converts 1/3 to 0.3333333/1. Maybe it's not worthwhile to reduce fractions?

Reply:
To track conversation IRL: Lets look into adding a reduce for fractions but not for reals (i.e. 6/2 -> 3 but 1/3 -> 1/3 and 0.5 -> 0.5).